File: ios/NHSOnline/NHSOnline/Data/ConfigurationResponse.swift
struct ConfigurationResponse {
    var callSuccessful: Bool
    var fidoServerUrl: String
    var isSupportedVersion: Bool
    var nhsLoginLoggedInPaths: [String]
    var knownServices: KnownServices

    init(_ callSuccessful: Bool = false,
         _ fidoServerUrl: String = "",
         _ isSupportedVersion: Bool = false,
         _ nhsLoginLoggedInPaths: [String] = [String](),
         _ knownServices: KnownServices = KnownServices()){
        self.callSuccessful = callSuccessful
        self.fidoServerUrl = fidoServerUrl
        self.isSupportedVersion = isSupportedVersion
        self.nhsLoginLoggedInPaths = nhsLoginLoggedInPaths
        self.knownServices = knownServices
    }
}

File: android/app/src/main/java/com/nhs/online/nhsonline/support/ConfigurationResponse.kt
package com.nhs.online.nhsonline.support

import com.nhs.online.nhsonline.services.knownservices.KnownServices

class ConfigurationResponse {
    var isSupportedVersion: Boolean = false
    var fidoServerUrl: String = ""
    var knownServices: KnownServices? = null
    var callSuccessful: Boolean = false
    var nhsLoginLoggedInPaths: List<String> = listOf<String>()
}